### ISSUE TITLE
Renders invisible colours when using dark theme

### ISSUE THREAD

**Issue by lonix1** [spectreconsole/spectre.console#930]
**Information**
 - OS: linux
 - Version: ubuntu 20
 - Terminal: gnome terminal
 - zsh 5.8
 - dotnet 6.0.400 with agnoster theme
 - Spectre.Console 0.44.0

**Describe the bug**
Some items are rendered in colours which cannot be seen in the terminal.

**To Reproduce**
- create project
  ````sh
  dotnet new console -o Demo
  cd Demo
  dotnet add package Spectre.Console
  ````
- Add code from the [getting started page](https://spectreconsole.net/cli/getting-started)
- `$ dotnet run -- --help`

**Expected behavior**
To be rendered so that I can see everything

**Screenshots**
![spectre](https://user-images.githubusercontent.com/40320097/187005503-7bf4a08a-a50e-48ad-b9b8-4cc30dc1e97e.jpg)

**Additional context**
I thought it could be zsh, or zsh's "agnoster" theme. So I ran it in dash without any theming:
````sh
sh
dotnet run -- --help
````
That produced the same output as above, so it's not the shell.

**Comment by lonix1** [comment]
I had an idea that maybe it's the terminal's theme. My `gnome-terminal` uses the extremely popular "Solarized Dark" theme. When I changed it to a light theme, I could see that text. So the conflict is definitely the terminal theme.

However, when I chose any other dark theme, I got the same problem. So the problem is not the theme itself, but rather that *it's a dark theme*.

Do the colours rendered by Spectre work nicely with dark themes?
